elastic/logs/challenges/logging-streams.json

```diff
@@ -0,0 +1,331 @@
+                    "ignore_malformed": false,
+                    "type": "date"
+                  },
+                  "dropped_attributes_count": {
+                    "type": "long"
+                  },
+                  "severity_number": {
+                    "type": "long"
+                  },
+                  "event_name": {
+                    "type": "keyword"
+                  },
+                  "attributes": {
+                    "type": "object",
+                    "subobjects": false
+                  },
+                  "scope.version": {
+                    "type": "keyword"
+                  },
+                  "resource.dropped_attributes_count": {
+                    "type": "long"
+                  },
+                  "scope.schema_url": {
+                    "type": "keyword"
+                  }
+                }
+              }
+            }
+        }
+      }
+    },
+    {
+      "name": "logs-pipeline",
+      "operation": {
+        "operation-type": "put-pipeline",
+        "id": "logs@stream.processing",
+        "body": {
+          "description": "Pipeline for processing logs stream",
+          "processors": [
+            {
+              "set": {
+                "description": "If '@timestamp' is missing, set it with the ingest timestamp",
+                "field": "@timestamp",
+                "override": false,
+                "copy_from": "_ingest.timestamp"
+              }
+            },
+            {
+              "pipeline": {
+                "name": "logs@json-pipeline",
+                "ignore_missing_pipeline": true
+              }
+            },
+            {
+              "dot_expander": {
+                "field": "*",
+                "ignore_failure": true
+              }
+            },
+            {
+              "normalize_for_stream": {}
+            },
+            {
+              "dot_expander": {
+                "path": "resource.attributes",
+                "field": "*",
+                "ignore_failure": true
+              }
+            },
+            {
+              "dot_expander": {
+                "path": "attributes",
+                "field": "*",
+                "ignore_failure": true
+              }
+            },
+            {
+              "script": {
+                "source": "ctx[\"stream.name\"] = params.field",
+                "lang": "painless",
+                "params": {
+                  "field": "logs"
+                }
+              }
+            },
+            {
+              "pipeline": {
+                "name": "logs@stream.reroutes",
+                "ignore_missing_pipeline": true
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "logs-index-template",
+      "operation": {
+        "operation-type": "create-composable-template",
+        "template": "logs@stream",
+        "body": {
+          "index_patterns": [
+            "logs"
+          ],
```

Comment:
This should match data streams created by the track (see [here](https://github.com/elastic/rally-tracks/blob/24c4b0b052a16758618e07c5159e20fffb69632c/elastic/logs/track.json#L181-L221)). Please change from `logs` to `logs-*`.